lib/modules/apostrophe-templates/index.js

```diff
@@ -696,5 +696,30 @@ module.exports = {
       req.data.aposBodyClasses = (req.data.aposBodyClasses ? (req.data.aposBodyClasses + ' ') : '') +
         bodyClass;
     };
+
+    // Add a body attribute to be emitted when the page is rendered. This information
+    // is attached to `req.data`, where `req.data.aposBodyDataAttributes` is built up
+    // using `name` as the attribute name which is automatically prepended with "data-"
+    // and the optional `value` argument
+    // Also receives a single object or an array of objects with name and value key pairs
+    // The default `outerLayoutBase.html` template outputs that string.
+
+    self.addBodyDataAttribute = function(req, name, value = '') {
+      var values = {};
+      if (_.isObject(name) && !_.isArray(name) && !_.isFunction(name)) {
```

Comment:
You can drop all checks here other than `_.isObject(name)` as the only allowed syntaxes now are the "name, value" syntax and the object syntax. Ditto the other places `_.isArray()` is still around, shouldn't need to do that.